docs/config/README.md

```diff
@@ -1250,7 +1250,7 @@ This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gc
 | Option            | Default                                                  | Description                                                      |
 | ----------------- | -------------------------------------------------------- | ---------------------------------------------------------------- |
 | `format`          | `'on [$symbol$account(@$domain)(\($region\))]($style) '` | The format for the module.                                       |
-| `symbol`          | `"☁️  "`                                                 | The symbol used before displaying the current GCP profile.       |
+| `symbol`          | `"☁️ "`                                                  | The symbol used before displaying the current GCP profile.       |
```

Comment:
Please don't add unrelated changes to the docs.

Reply:
Thank you for your comment.
This is an unintended change.
I corrected.